### PR TITLE
PMM-4033 Remove `oneof` from requests.

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

File: api/inventorypb/agents.proto [hunk 1]
```diff
@@ -318,14 +317,12 @@ message ChangeCommonAgentParams {
...
 message ListAgentsRequest {
-  oneof filter {
-    // Return only Agents started by this pmm-agent.
-    string pmm_agent_id = 1;
-    // Return only Agents that provide insights for that Node.
-    string node_id = 2;
-    // Return only Agents that provide insights for that Service.
-    string service_id = 3;
-  }
+  // Return only Agents started by this pmm-agent.
```
Comment (BupycHuk) [inline api/inventorypb/agents.proto]: let's add comment that only one of them might be used.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-438/

File: api/inventorypb/agents.proto (hunk 1 above)
Comment (AlekSi) [reply api/inventorypb/agents.proto]: Done

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-438/

**Pull request merged**